opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -26,6 +26,9 @@ impl UserEventsExporter {
     /// Create instance of the exporter
     pub(crate) fn new(provider_name: &str) -> Result<Self, String> {
         // Validate provider_name
+        if provider_name.is_empty() {
```

Comment:
please add this to changelog too. I also suggest to do separate PRs to keep scope of PR short and focused.